core/math/src/main/java/org/arakhne/afc/math/geometry/d2/afp/BasicPathShadow2afp.java

```diff
@@ -503,7 +503,7 @@ private void crossSegmentShadowLine(
      * @mavengroupid $GroupId$
      * @mavenartifactid $ArtifactId$
      */
-    private final  class CoordinatesParam {
```

Comment:
Why have you introduced this class? Why not using standard function argument passing?